tests/snuba/api/endpoints/test_organization_events_stats_span_indexed.py

```diff
@@ -747,18 +747,38 @@ def test_extrapolation_count(self):
         assert response.status_code == 200, response.content
         data = response.data["data"]
         meta = response.data["meta"]
-        confidence = meta["accuracy"]["confidence"]
         assert len(data) == 6
         assert response.data["meta"]["dataset"] == self.dataset
 
         for expected, actual in zip(event_counts, data[0:6]):
             assert actual[1][0]["count"] == expected * 10
 
+        accuracy = meta["accuracy"]
+        confidence = accuracy["confidence"]
+        sample_count = accuracy["sampleCount"]
+        sample_rate = accuracy["samplingRate"]
         for expected, actual in zip(event_counts, confidence[0:6]):
             if expected != 0:
-                assert actual["count()"] == "low"
+                assert actual["value"] == "low"
             else:
-                assert actual["count()"] is None
+                assert actual["value"] is None
+
+        # Check old confidence format, TODO: remove this once frontend is updated
+        confidence = response.data["confidence"]
+        for expected, actual in zip(event_counts, confidence[0:6]):
+            if expected != 0:
+                assert actual[1][0]["count"] == "low"
+            else:
+                assert actual[1][0]["count"] is None
```

Comment:
Is this returning it without the parenthesis now? Might break the frontend if this is changed.

Reply:
Gahhh, i hate this format, good catch